scripts/github_action.py

```diff
@@ -0,0 +1,61 @@
+class GithubActionsTransformer():
+    def __init__(self, name):
+        super(GithubActionsTransformer, self).__init__()
```

Comment:
What does calling `super` do here? AFAICT there's no base class?

Reply:
good catch, I had a base class that felt redundant but forgot to clean up

Reply:
fixed